### ISSUE TITLE
Enhancement: All the icons for ageng are the same in the list.

### ISSUE THREAD

**Issue by Passerby1011** [danny-avila/LibreChat#4515]
### What features would you like to see added?

I hope that in the conversation list, the agents are displayed with the dialogue assistant's icon.

### More details

All the agents in the conversation list display the same icon, but there are different assistants within the agents, and each assistant can be set with a different icon.

### Pictures

![image](https://github.com/user-attachments/assets/5058548c-7410-46d5-bf30-61e8e248750e)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by cjccjj** [comment]
yeah. Same for Assistants. I wish for this too. It has been too complicated that each type of endpoint requries different logic. 
the modelSpecs can be further optimized to act as uniform access point, with icons, etc. 
as also mentioned here: https://github.com/danny-avila/LibreChat/discussions/4492
